CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0
 `Unreleased`_
 =============
 
+Fixed
+-----
+* Fixed `get_services()` fails because no GattSession on WinRT backend.
```

Comment:
Since the code changes were revised, I think this should be something like

```rst
Changed
---------
* ``BleakClient`` methods now raise ``BleakError`` if called when not connected in WinRT backend.